pkgs/development/ocaml-modules/luv/default.nix

```diff
@@ -6,12 +6,12 @@
 
 buildDunePackage rec {
   pname = "luv";
-  version = "0.5.11";
+  version = "0.5.12";
   useDune2 = true;
```

Comment:
```suggestion
```

Reply:
You may want to set `minimalOCamlVersion` too (`luv` wont build with OCaml 4.02).

Reply:
Since it depends on `fmt` we in reality can't use anything earlier than 4_08 so I set that as the `minimalOcamlVersion`. I did the changes requested and this builds fine: `nix build .#ocaml-ng.ocamlPackages_{5_0,5_1,4_08,4_12,4_14}.luv`

Reply:
No, it does not depend on `fmt`. It used to build fine for OCaml 4.03–4.07.

Reply:
(but if earlier versions are not used in `nixpkgs`, that’s probably fine)

Reply:
The change to doCheck screwed me over, I reverted it and added a comment